src/group/communication/app.js

```diff
@@ -1,9 +1,11 @@
 import {
 	MOCK_GROUP_COMMUNICATION,
-	MOCK_GROUP_COMMUNICATION_COMMENTS,
 	MOCK_GROUP_COMMUNICATION_FOLLOWERS,
 	MOCK_GROUP_COMMUNICATION_SUGGESTED_FOLLOWERS
 } from './api';
+import {
+	MOCK_COMMENTS_LIST,
+} from './../api';
```

Comment:
The build for this repo is just a Babel transpile - no code is actually executed.

I guess we probably should add some kind of basic unit test, but import errors are currently silent.